lib/navigation/imu_displacement.cpp

```diff
@@ -6,20 +6,20 @@
 
 namespace hyped::navigation {
 
-ImuDisplacement::ImuDisplacement(const core::ITimeSource &time)
+ImuDisplacement::ImuDisplacement(const core::ITimeSource &time, const core::TimePoint initial_time)
```

Comment:
why do we want to have this as a parameter rather than taking `time.now()`?

Reply:
Because it is the timepoint of a measurement (class will take the timestamp of the first accelerometer reading as a constructor argument). The time difference between these should be very minimal but when we're working at nanosecond precision and process accelerometer data as heavily as we do, we probably shouldn't ignore this difference?

Reply:
surely this wants to be constructed during initialisation, i.e. when we're not moving anyways?

in that case, we're talking about seconds before we actually end up moving, no? if this is absolutely necessary, which it may very well be, find a way to make this intention clear in the types (not in comments!)

Reply:
I guess it depends on how/when this is called. I'll have to consider when this is constructed and how this will fit in with accelerometer calibration.